<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpsInfo</class>
 <widget class="QWidget" name="GpsInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="gpsLatValue">
       <property name="text">
        <string>0.0 °</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="gpsLonLabel">
       <property name="text">
        <string>Longitude:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="gpsSigLabel">
       <property name="text">
        <string>Signal:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="gpsAltValue">
       <property name="text">
        <string>0.0 m</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="gpsAltLabel">
       <property name="text">
        <string>Altitude:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="gpsRxLabel">
       <property name="text">
        <string>Receiver:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="gpsLonValue">
       <property name="text">
        <string>0.0 °</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="gpsRxValue">
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="gpsLatLabel">
       <property name="text">
        <string>Lattitude:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="gpsFixLabel">
       <property name="text">
        <string>Fix:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="gpsSigValue">
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="gpsFixValue">
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
